tests/unit/amo/components/ErrorPage/TestNotFound.js

```diff
@@ -35,26 +31,26 @@ describe(__filename, () => {
   it('renders a not found error', () => {
     const root = render();
 
-    expect(root.find('.ErrorPage')).toHaveProp('header', 'Page not found');
-    expect(root.find(SuggestedPages)).toHaveLength(1);
-    expect(root.find('.NotFound-fileAnIssueText').html()).toContain(
-      'file an issue',
-    );
-  });
-
-  it('renders a disabled by developer error', () => {
-    const root = render({ errorCode: ERROR_ADDON_DISABLED_BY_DEV });
-
-    expect(root.find('.NotFound-explanation').html()).toContain(
-      'This add-on has been removed by its author.',
+    expect(root.find('.ErrorPage')).toHaveProp(
+      'header',
+      'Oops! We can’t find that page',
     );
-  });
 
-  it('renders a disabled by admin error', () => {
-    const root = render({ errorCode: ERROR_ADDON_DISABLED_BY_ADMIN });
-
-    expect(root.find('.NotFound-explanation').html()).toContain(
-      'This add-on has been disabled by an administrator.',
-    );
+    // First paragraph with a link to GitHub.
```

Comment:
nit: should these comments be complete sentences?

Reply:
yep